codemods/use-i18n-hook.js

```diff
@@ -0,0 +1,128 @@
+// Press ctrl+space for code completion
+
+export default function transformer(file, api) {
+  const j = api.jscodeshift
+  const root = j(file.source)
+
+  const isI18nProp = prop => {
+    return prop.key && (prop.key.name === 't' || prop.key.name === 'f')
```

Comment:
There's also the `lang` prop that is injected by translate, it would be good to migrate it as well.

Reply:
It was there originally but I saw than in Banks it would migrate by error a component that was not included in the I18n provider (the top level App component, that takes a lang). So I removed it.

I could maybe make this dynamic so that the user of the codemod can choose whether to migrate components using lang.

Reply:
Yep I think that would be great! Otherwise I think we risk ending up with leftover `lang` props that won't work.

Reply:
> Otherwise I think we risk ending up with leftover lang props that won't work.

No, we will just end up with components using `lang` that will just not use useI18n and stay with translate(). This is not a big problem IMHO.